pkg/appmanager/appManager.go

```diff
@@ -1199,6 +1221,31 @@ func (appMgr *Manager) setServerSslProfile(
 	rsCfg *ResourceConfig,
 	route *routeapi.Route,
 ) {
+	// First handle the Default for SNI profile
+	if appMgr.routeConfig.ServerSsl != "" {
+		// User has provided a name
+		profile := ProfileRef{
+			Name:      appMgr.routeConfig.ServerSsl,
```

Comment:
Idiomatic go prefers acronyms with all caps, ServerSSL, this name may cause extra linter warnings.